samples/openapi3/server/petstore/springboot-nullable-disabled/src/main/java/org/openapitools/model/Category.java

```diff
@@ -0,0 +1,105 @@
+package org.openapitools.model;
+
+import java.net.URI;
+import java.util.Objects;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.time.OffsetDateTime;
+import javax.validation.Valid;
+import javax.validation.constraints.*;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+
+import java.util.*;
+import javax.annotation.Generated;
+
+/**
+ * A category for a pet
+ */
+
+@Schema(name = "Category", description = "A category for a pet")
+@Generated(value = "org.openapitools.codegen.languages.SpringCodegen")
+public class Category   {
+
+  @JsonProperty("id")
+  private Optional<Long> id;
+
+  @JsonProperty("name")
+  private Optional<@Pattern(regexp = "^[a-zA-Z0-9]+[a-zA-Z0-9\\.\\-_]*[a-zA-Z0-9]+$") String> name;
```

Comment:
I see.
So far, optionals have only been used as parameters. I think for member variables we need a new mustache without bean validation. Because the beanvalidation is in <> always makes it awkward.

Reply:
Created `optionalDataTypeWithoutAnnotations` - I did leave it on the setter but I'm not that familiar with how the validation is applied. I can convert it to getter only if that's more correct.

Reply:
Bean validation for DTOs/Pojos have to be set on getter or fields. Currently the template set it on getter